docs/release-notes/release-notes-0-17-17.md

```diff
@@ -89,6 +89,9 @@ The stream stats API endpoint `/_fabric/{fabric}/_api/streams/{stream}/stats` no
 
 Description of update.
 
+### Connection Metrics API Endpoint
+
+You can now access the `GET /_api/connections/metrics` endpoint to retrieve metrics for your connections.
```

Comment:
@diana-macrometa Response will include metrics in following response.
```json
[
    {
      "metric": "Ingested Documents",
      "data": []
    },
    {
      "metric": "Exported Documents",
      "data": []
    },
    {
      "metric": "Ingested Bytes",
      "data": []
    },
    {
      "metric": "Exported Bytes",
      "data": []
    },
    {
      "metric": "Ingest Lag",
      "data": []
    },
    {
      "metric": "Export Lag",
      "data": []
    },
    {
      "metric": "Ingest Errors",
      "data": []
    },
    {
      "metric": "Export Errors",
      "data": []
    },
    {
      "metric": "Successful Workflows",
      "data": []
    },
    {
      "metric": "Failed Workflows",
      "data": []
    }
  ]
```

Reply:
Updated, please take a look.